### DISCUSSION TITLE
360º Object Movies

### DISCUSSION THREAD

**Discussion by ChrisHannan-zl** [openseadragon/openseadragon#2232] [Ideas]
Hello all,

Has anyone ever created or attempted to create an Object Movie capability to Open SeaDragon? 
Is a 360º object even possible to create from 72 or 180 images presented together as an image sequence and "rotatable" with click and drag or swipe?

**Comment by iangilman** [comment]
I would say it's definitely possible! Here's an example of something like that (without the interaction):

https://codepen.io/iangilman/pen/PjdyGv

Adding the dragging shouldn't be difficult. Beyond that, the big thing you'll have to pay attention to is making sure all the images get loaded in a timely fashion, and that the cache is big enough that they all stay loaded.
